.github/workflows/push.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
          mkdir -p ~/.docker
          echo '{"experimental": "enabled"}' >> ~/.docker/config.json
-         echo "IMAGE_EXISTS=$(docker manifest inspect ghcr.io/${{ github.repository }}-${{ matrix.tennes }}:${{ github.sha }} > /dev/null && echo 1 || echo 0)" >> $GITHUB_ENV
+         echo "IMAGE_EXISTS=$(docker manifest inspect ghcr.io/${{ github.repository }}-${{ matrix.tennens }}:${{ github.sha }} > /dev/null && echo 1 || echo 0)" >> $GITHUB_ENV
```

Comment:
Fix my own spelling error.
Found out as i view an rerun of an action where the image where build again